infra/openshift-manifests/reply-service/service.yaml

```diff
@@ -66,6 +68,10 @@ spec:
     - protocol: TCP
       port: 80
       targetPort: 8080
+  sessionAffinity: ClientIP
+  sessionAffinityConfig:
+    clientIP:
+      timeoutSeconds: 3600
```

Comment:
what is this needed?

Reply:
I want the client to talk to the same pod.

Maybe not so necessary right now, as we only have a single connection.

Reply:
Removed